ui/css/design-system/_colors.scss

```diff
@@ -16,12 +16,10 @@ $color-map: (
   'primary-alternative': --color-primary-alternative,
   'primary-muted': --color-primary-muted,
   'primary-inverse': --color-primary-inverse,
-  'primary-disabled': --color-primary-disabled,
```

Comment:
No instances of primary-disabled exists

![Screenshot 2024-05-31 at 4 28 46 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/9e4a0735-cfa9-466a-b09c-0b8907b09209)